### PR TITLE
fix/padatious_mess

### PR DESCRIPTION
**Pull request by JarbasAl**
intent service code for padatious is duplicated

- this was probably the cause of #2786
- new commits seem to be maintaining both files instead of deleting old location? https://github.com/MycroftAI/mycroft-core/commit/8a1ee0e106d10dc008300945ccbefea70ac90149#diff-8df1dc1f3977c3ffcdefe4c24709f1c67374caa3863e4e0caf04f1b652603671

this PR removes old location

### CODE REVIEW

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-2843))

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-2843))

**Comment by forslund** [comment]
Thanks Jarbas, looks good. You can probably remove the old location completely now that the next version should be 21.02 it can be dropped.

File: mycroft/skills/intent_services/padatious_service.py [hunk 1]
```diff
@@ -52,6 +52,11 @@ def __init__(self, bus, config):
         self.bus.on('detach_intent', self.handle_detach_intent)
         self.bus.on('detach_skill', self.handle_detach_skill)
         self.bus.on('mycroft.skills.initialized', self.train)
+        self.bus.on('intent.service.padatious.get', self.handle_get_padatious)
+        self.bus.on('intent.service.padatious.manifest.get',
+                    self.handle_manifest)
+        self.bus.on('intent.service.padatious.entities.manifest.get',
+                    self.handle_entity_manifest)
```
Comment (forslund) [inline mycroft/skills/intent_services/padatious_service.py:59]: Don't these duplicate the bus handlers registered in [intent_service.pyL129](https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/skills/intent_service.py#L129)?

They should be removed as well if these are moved here.

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-2843))

**Review by forslund — APPROVED** [review]
Nice, glad to be rid of it :)

**Pull request merged**